### DISCUSSION TITLE
Project References

### DISCUSSION THREAD

**Discussion by Chevelleboyer** [dudykr/stc#1054] [General]
Hello 👋 

Sorry if this is obvious to everyone else, but will `stc` support [project references](https://www.typescriptlang.org/docs/handbook/project-references.html)?

My team plans to one day use `stc` ❤️ We recently enabled project references and saw some significant speed gains. When this is ready, can it be used alongside project references? If not, do you anticipate `stc` being faster than `tsc` in conjunction with project references? 👀

**Comment by kdy1** [comment]
I think `stc` without project references will be faster than `tsc` with it (because the single core restriction is way too much restriction for nowadays), but yes, `stc` will support project references.
